types/cloudinary-core.d.ts

```diff
@@ -434,6 +434,18 @@ export class PictureTag extends HtmlTag {
     static "new"(name: string, publicId: string, options?: Transformation.Options): PictureTag;
 }
 
+/**
+ * Creates an HTML (DOM) Image tag using Cloudinary as the source.
+ * @constructor SourceTag
+ * @extends HtmlTag
+ * @param {string} [publicId]
+ * @param {Object} [options]
+ */
+export class SourceTag extends HtmlTag {
+    static "new"(publicId: string, options?: Transformation.Options): SourceTag;
+    static "new"(name: string, publicId: string, options?: Transformation.Options): SourceTag;
+}
```

Comment:
@eyalktCloudinary , how do I use this sourceTag? in pictureTag? in videoTag?

Reply:
A SourceTag instance is returned by sourceTag() ([cloudinary.js](https://github.com/cloudinary/cloudinary_js/blob/42f9795a6f3a624ba0d3565eb8cecd30f4a55e19/src/cloudinary.js#L291)), though I couldn't find any invocations of this function. The client that has raised this issue has found the function in our code and tries to use it, but couldn't - https://github.com/cloudinary/cloudinary_js/issues/189

Reply:
I'm guessing we're supposed to use it in pictureTag to create 
`<picture><source></source></picture>`
but it seems that we're not

Reply:
@eyalktCloudinary , let's check how this is used before we merge the PR

Reply:
I think this is an internal class was exposed(alone and as a builder), but was never documented.

We use `new SourceTag()` in pictureTag, but I don't think we ever meant for the sourceTag itself to work in isolation, and for it to be public.